docs/source/installation/worker.rst

```diff
@@ -115,17 +115,21 @@ Install dependencies for SMTP/SMTPS check
   chmod a+x /usr/bin/smtps_check
 
 Install dependencies for NFS check
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 ::
 
   apt-get install -y libnfs-dev
   source /home/engine/scoring_engine/env/bin/activate && pip install -I "libnfs==1.0.post4"
 
+Install dependencies for Telnet check
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+::
+
+  source /home/engine/scoring_engine/env/bin/activate && pip install -I "telnetlib3==1.0.1"
```

Comment:
Do we want to install this in the worker docker container as well?

Reply:
So I'm not sure if we want the worker to be able to handle any check that we could possible enable for it. Otherwise people would have to edit the worker docker if they wanted to install a new check.

Reply:
Fixed